webapp/publisher/snaps/logic.py

```diff
@@ -50,6 +50,29 @@ def get_snaps_account_info(account_info):
     return user_snaps, registered_snaps
 
 
+def get_snap_names_by_ownership(account_info):
+    """Get list of snaps names user is collaborator of
```

Comment:
of what!!! the mystery  isn't resolved

Reply:
of snap names... no mystery

Reply:
@Lukewh So it is correct English? I wasn't sure when writing it as well ;)

Reply:
The only thing I'd change is "snaps" > "snap" as the "names" pluralises...

Reply:
@Lukewh but it's a list of "names" not snaps... so.. "list of snaps' names ..." maybe?